src/main/java/org/commcare/formplayer/beans/menus/PeristentCommand.java

```diff
@@ -0,0 +1,47 @@
+package org.commcare.formplayer.beans.menus;
+
+import org.checkerframework.checker.units.qual.A;
+import org.commcare.modern.session.SessionWrapper;
+import org.commcare.suite.model.MenuDisplayable;
+
+import java.util.ArrayList;
+
+import lombok.EqualsAndHashCode;
+
+/**
+ * Model used to represent a persistent menu in FP HTTP API
+ */
+@EqualsAndHashCode
+public class PeristentCommand {
```

Comment:
nit: typo, should be `PersistentCommand`

Reply:
Thanks, corrected in https://github.com/dimagi/formplayer/pull/1594/commits/bef10e8f9ae9e10d00d5a797518ca1e4efe143fd